test/submission.e2e.js

```diff
@@ -126,6 +130,12 @@ test('Happy path', async t => {
     .eql(manuscript.title)
     .expect(dashboard.stages.textContent)
     .eql('QA')
```

Comment:
Do we perhaps want this to change to "Exporting to eJP", until the promise has resolved & then show "QA" afterwards? That way we would be making the difference between finishing the submission wizard & finishing the submission (including getting it into eJP) more obvious to our users. At the moment we're just relying on them seeing a failure email, to let them know if it hasn't been successful, but our dashboard itself gives them a false sense of security that all is well.

@hdrury1, I assume all of this has been considered in the dashboard designs & that's a different epic. But just highlighting it here in case?

Reply:
Yes, it definitely shouldn't be "QA" when the stage is called Quality Check 😀 

A status of 'Submitting' followed by 'Submitted' would probably make most sense to users. It's very unlikely a user would see the 'Submitting' stage though as it will only last a second or two at most to be followed by either 'Submitted' or 'Failed'. 

Just to be clear though, the failure email would go to staff, not authors.

Reply:
Hahaha. Yeah all makes total sense.

I know the dashboard designs are still pending & this version is just a dummy. Was just wondering whether it was worth amending that now, if nothing else, just as a reminder to ourselves? Since the "QA" thing is actually in our e2e tests as well as sitting on the dashboard page.

Anyway, not necessarily important. I'm mostly pointing this stuff out as a reminder to ourselves. Happy for all this to be merged now :+1: